### PR TITLE
Update imagery from city of Bern

### PR DESCRIPTION
**Pull request by habi**
- Removed Stadt_Bern_10cm-2016-WMS.geojson and Stadt_Bern_10cm-2020-WMS.geojson, as "Dienst eingestell", e.g. discontinued
- Added Stadt_Bern-Orthofotos-2023-WMTS.geojson, is an updated copy of Stadt_Bern_10cm-2020-WMS.geojson

### CODE REVIEW

File: sources/europe/ch/Stadt_Bern-Orthofotos-2023-WMTS.geojson [hunk 1]
```diff
@@ -3,25 +3,24 @@
...
+        "name": "Stadt Bern 10cm (2023)",
         "available_projections": [
             "CRS:84",
             "EPSG:2056",
             "EPSG:3857",
             "EPSG:4326",
             "EPSG:21781"
         ],
-        "url": "https://map.bern.ch/wms/OpenData/proxy.php?LAYERS=luftbild_2020&STYLES=default&FORMAT=image/jpeg&CRS={proj}&WIDTH={width}&HEIGHT={height}&BBOX={bbox}&VERSION=1.3.0&SERVICE=WMS&REQUEST=GetMap",
+        "url": "https://map.bern.ch/arcgis/services/Geoportal/Orthofoto_2023/MapServer/WmsServer?FORMAT=image/png&TRANSPARENT=TRUE&VERSION=1.3.0&SERVICE=WMS&REQUEST=GetMap&LAYERS=0&STYLES=&CRS={proj}&WIDTH={width}&HEIGHT={height}&BBOX={bbox}",
```
Comment (arch0345) [inline sources/europe/ch/Stadt_Bern-Orthofotos-2023-WMTS.geojson:22]: This looks like the URL formatting for a WMS endpoint, not WMTS

**Review by arch0345 — CHANGES_REQUESTED** [review]

File: sources/europe/ch/Stadt_Bern-Orthofotos-2023-WMTS.geojson (hunk 1 above)
Comment (habi) [reply sources/europe/ch/Stadt_Bern-Orthofotos-2023-WMTS.geojson:22]: I copy-pasted the wrong line from https://josm.openstreetmap.de/wiki/Maps/Switzerland#StadtBern10cm2023

Does ELI offer a 'mirror' option?

**Comment by habi** [comment]
Fixes https://github.com/osmlab/editor-layer-index/issues/2365 if accepted

File: sources/europe/ch/Stadt_Bern-Orthofotos-2023-WMTS.geojson (hunk 1 above)
Comment (arch0345) [reply sources/europe/ch/Stadt_Bern-Orthofotos-2023-WMTS.geojson:22]: > Does ELI offer a 'mirror' option?

Not that I know of, however it looks like #816 would add support for this

**Review by arch0345 — APPROVED** [review]

**Comment by arch0345** [comment]
@grischard if you have the time, could you please review this PR? Thanks

**Pull request merged**